react-frontend/src/components/Learning/podcast.js

```diff
@@ -0,0 +1,107 @@
+import React, { useContext } from 'react';
+import { Col, Row } from 'react-flexbox-grid';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons';
+import dateFormat from 'dateformat';
+
+import BannerSideImage from '../PageElements/Banners/bannerSideImage';
+import PodcastBanner from './podcastBanner';
+import PodcastCollapsedMenu from './podcastCollapsedMenu';
+
+import Query from '../Query';
+import PODCASTS_QUERY from '../../queries/learning/podcasts';
+import { AppConfigContext } from '../../providers/AppConfig';
+
+import { ContentBlockContainer } from '../StyleComponents/pageContent';
+import { SubHeading, SubSubHeading } from '../StyleComponents/headings';
+import { HrefLinkStandalone } from '../StyleComponents/htmlTags';
+
+const podcastImage = require('../../images/pngIllustrations/podcastGray.png')
+  .default;
+
+function Podcast() {
+  const strapiMediaUrl = useContext(AppConfigContext)['state'][
+    'strapiMediaUrl'
+  ];
+  return (
+    <div>
+      <PodcastBanner />
+      <ContentBlockContainer id="courses">
+        <Row>
+          <Col sm={12}>
```

Comment:
The "View all" isn't vertically aligned with the "Recent Episodes" heading:

![vertical-alignment](https://user-images.githubusercontent.com/16946297/124505388-40efb080-dd7e-11eb-8033-0601bc954637.png)

I found I could get a nice alignment by: 

1. To the `<Col>`:  `style={{ alignItems: 'baseline', display: 'flex', justifyContent: 'space-between', marginBottom: '16px' }}`
2. To the `<SubHeading>`:  `style={{ display: 'inline-block', margin: '0' }}`
3. To the `<HrefLinkStandalone>`:  `style={{ float: 'right', margin: '0' }}`

![Screen Shot 2021-07-05 at 10 55 29 AM](https://user-images.githubusercontent.com/16946297/124506082-c7f15880-dd7f-11eb-8a60-3b6f859c4c7f.png)

Reply:
That works.